src/IsotonicFPOP.cpp

```diff
@@ -1,23 +1,24 @@
 /* -*- compile-command: "R CMD INSTALL .." -*- */
-  
-  #include <vector>
-  #include <stdio.h>
-  #include "funPieceListLog.h"
-  #include <math.h>
-  
-  void IsotonicFPOP
-(int *data_vec, double *weight_vec, int data_count,
```

Comment:
I guess you don't need weights for your neuro data application? Even if you don't need them right now, you could keep them in the implementation, and use weight_i=1 for all i.